torch/csrc/jit/codegen/cuda/lower_utils.cpp

```diff
@@ -208,6 +208,66 @@ kir::Expr* applyReplacements(
   }
 }
 
+c10::optional<IterDomain*> getMaybeWarpReductionDim(
+    const kir::ReductionOp* node) {
+  kir::TensorView* kir_tv = nullptr;
+  auto kir_ti = dynamic_cast<kir::TensorIndex*>(node->out());
+  if (kir_ti) {
+    kir_tv = kir_ti->view();
+  } else {
+    kir_tv = dynamic_cast<kir::TensorView*>(node->out());
+  }
```

Comment:
nit: `ir_utils::getTVOutput` can be used.

Reply:
Updated. Thanks!